Dapper NET40/SqlMapper.cs

```diff
@@ -89,11 +90,13 @@ internal static CommandDefinition ForCallback(object parameters)
 
         internal void OnCompleted()
         {
-            if (parameters is SqlMapper.IParameterCallbacks)
+            var parametersCallbacks = parameters as SqlMapper.IParameterCallbacks;
+            if (parametersCallbacks != null)

```

Comment:
this isn't a clear improvement; although it may feel intuitive that the "as" only does the thing once, in reality neither is clearly better (and often the is/not-null can be more efficient) - feels like yak shaving